public/gallery.html

```diff
@@ -262,5 +262,6 @@
 
 <script src="JavaScript/common.js"></script>
 <script src="JavaScript/gallery.js"></script>
```

Comment:
Check out https://www.sitepoint.com/using-es-modules/. Using ES modules inside the browser means that you could add only the relevant script here and then import anything else needed in the file. 
What do you think? Would you like to make this change in a separate PR? I will leave it up to you